_posts/2025-03-26-kubeflow-1.10-release.md

```diff
@@ -0,0 +1,276 @@
+provides a user-friendly web interface for managing machine learning models within the Kubeflow platform. It centralizes
+model metadata, version tracking, and artifact management, streamlining MLOps workflows.
+
+Key features include:
+
+* Easy model registration with custom metadata
+* Comprehensive model management with filtering and sorting
+* Archiving capabilities
+* Version control
+* Metadata editing
+
+![Model Registry UI](../images/2025-03-26-kubeflow-1.10-release/model-registry-ui.png)
+
+The UI interacts with the Model Registry's REST API, making it accessible to users of all technical backgrounds and
+enhancing collaboration across data science, ML engineering, and MLOps teams.
+
+To get started with the Model Registry UI, which is currently in Alpha, you can follow the instructions
+[here](https://www.kubeflow.org/docs/components/model-registry/installation/#installing-on-kubeflow-platform).
+
+The Kubeflow Model Registry UI Team would like to conduct user research to identify possible enhancements we can contribute in future iterations of the Kubeflow Model Registry UI. If you are interested in participating in this study, please fill out [this survey](https://docs.google.com/forms/d/e/1FAIpQLSeCveL-b0NyUohYa86I3VeTXeynEQLpV5Loj-1HkoUVDwlVCQ/viewform).
+### Custom Storage Initializer
+
+The Model Registry Custom Storage Initializer (CSI) is a custom implementation of the KServe ClusterStorageContainer.
+This feature allows users to utilize Model Registry metadata to download and deploy models efficiently. With the newest
+release of the Model Registry, it is now possible to install and use the Custom Storage Initializer (CSI).
+
+You can find detailed installation instructions and a small example in the "Getting Started" section of the Model
+Registry component on the Kubeflow website.
+
+For additional information and future developments towards better integration with KServe, you can refer to the slides
+[here](https://docs.google.com/presentation/d/1wprxN0n23EMkPRX_PaZZcIzZbn_i8Sh_).
+
+## Training Operator (Trainer) & Katib
+
+Kubeflow 1.10 enhances the Training Operator and Katib, providing new tools and APIs for hyperparameter optimization,
+particularly for large language models.
+
+Moreover, the Kubeflow Training Operator now supports [JAX for distributed training](https://github.com/kubeflow/trainer/issues/1619),
+enabling users to leverage JAX's capabilities for efficient and scalable model training.
+
+Finally, if you want to get involved with Trainer V2, take a look at this [KEP](https://github.com/kubeflow/trainer/tree/master/docs/proposals/2170-kubeflow-trainer-v2)
+and [issue](https://github.com/kubeflow/trainer/issues/2170).
+
+### Hyperparameter Optimization API for LLMs
+
+Kubeflow 1.10 introduces a new high-level [API for hyperparameter tuning](https://github.com/kubeflow/katib/issues/2339),
```

Comment:
can we mention katib instead of kubeflow 1.10, since we are talking about each specific component.
cc @andreyvelich